src/api/index.ts

```diff
@@ -36,14 +37,45 @@ export const Responses = {
   }
 }
 
+export type SIGNALK_API_ID =
+  | 'resources'
+  | 'course'
+  | 'history'
+  | 'autopilot'
+  | 'anchor'
+  | 'logbook'
+
+export const isSignalKApi = (id: SIGNALK_API_ID): boolean => {
+  return [
+    'resources',
```

Comment:
The rabbit hole goes deeper: would it not be important to get a list of active resource providers?

Reply:
I think this is going too deep as the intent is to provide a high level feature list for apps to adapt their UI, etc to the capability available on the server.

Where this is not an API, the presence of a plugin is generally enough to determine if the capability exists.